scripts/utils/InsertNuGetIntoSdkAndCli.ps1

```diff
@@ -1,17 +1,17 @@
 ﻿<#
 .SYNOPSIS
-Script to insert NuGet into CLI and SDK. 
```

Comment:
Consider rephrasing to `Script to insert NuGet into dotnet projects: CLI, SDK and TOOLSET`

Reply:
fixed.